src/frontend/js/components/SearchFilterGroupModal/index.tsx

```diff
@@ -22,6 +22,14 @@ const messages = defineMessages({
   },
 });
 
+// The `setAppElement` needs to happen in proper code but breaks our testing environment.
+// This workaround is not satisfactory but it allows us to both test <SearchFilterGroupModal />
+// and avoid compromising accessibility in real-world use.
+const isTestEnv = !!jest;
```

Comment:
This is not working, I have an error in my browser: `ReferenceError: jest is not defined`